docs/instructions.md

```diff
@@ -221,46 +220,45 @@ PREDICT_MODULE=finance_income_prediction.predict
 
-here:
+NOTE: In the case of using the daemon s2i builder you only need to specify `ROUTES`.
+`PREDICT_MODULE` and `PREDICT FUNCTION` are optional when using the `s2i` daemon builder image, though it is used in all examples for readability.
 
-`adult_income_rf/predict` is the web-service route
-`finance_income_prediction.predict` is the python module in which predict functionality is defined
-`predict_adult_income_rf` is the predict function name defined in the `finance_income_prediction.predict` module shown above
+**Example:**
 
+`adult_income_rf/predict` is the web-service route.
+`finance_income_prediction.predict` is the python module in which predict functionality is defined.
+`predict_adult_income_rf` is the predict function name defined in the `finance_income_prediction.predict` module.
 
-## all
 
-All models can be trained and packaged together as well, exposing predict for an entire project/usecase(with 3 models here) as one single container with multiple routes for each of those models.
+## All
 
-> note `all` creates a new predict container with all routes in exactly the same way as individual models are created. Hence it needs the same files, some of which like .s2i/environment can be auto generated
+Models can be trained and packaged together, exposing predict for an entire project or use case as one single container with multiple routes for each of those models.(exemplified with the three models here)
 
+**NOTE**: `all` creates a new predict container with all routes in exactly the same way individual models are created. Hence it needs the same files, some of which can be auto generated, like in .s2i/environment.
 
-`training` here just invokes the individual model's train functionality. no sepate train module needed.
+`training` invokes each individual model's train functionality. Separate train modules are not necessary.
 
-`prediction` here needs a predict module with just the correct function imports from individual model dirs to invoke the correct method when called with that endpoint. (TODO: automate predict.py using code gen)
+`prediction` requires a predict module with the correct function imports from individual model dirs to invoke the correct method when called with that endpoint
 
-`requirements.txt`  list of pip installable modules
-required when creating single container with all models
+`requirements.txt` list of pip installable modules required when creating a single container with all models
 
-Here the `.s2i/environment` is auto-generated from individual model's .s2i/environment files defined inside each of the container
+The `.s2i/environment` is auto-generated from individual model's `.s2i/environment` files defined inside each of the containers
 
-Here user needs to specify:
+**EXAMPLE:**
+The user must specify:
 
-1.  MODEL_DIRS := xgBoost logisticRegression randomForest
+- `MODEL_DIRS` = xgBoost logisticRegression randomForest
```

Comment:
Should the quotes be around the whole clause (although in this case you probably need to keep the :=)? Is this intended to be exactly what's in the Makefile? https://github.com/CognitiveScale/certifai-reference-models/blob/smm-documentation-review-2/finance_income_prediction/all/Makefile